### PR TITLE
18744 Tornado: Add options to change font-size/color/position for Group labels

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -71,7 +72,7 @@ module.exports = (config) => {
         ],
         preprocessors: {
             [recursivePathToTests]: ['typescript'],
-            [srcRecursivePath]: ['sourcemap', 'coverage']
+            [srcRecursivePath]: ['sourcemap']
```
Comment (ignatvilesov) [inline karma.conf.js]: Please turn the coverage on again.

File: package.json [hunk 2]
```diff
@@ -36,15 +37,15 @@
...
-    "powerbi-visuals-utils-chartutils": "0.3.0",
-    "powerbi-visuals-utils-colorutils": "0.2.1",
-    "powerbi-visuals-utils-dataviewutils": "1.0.0",
-    "powerbi-visuals-utils-formattingutils": "1.0.0",
-    "powerbi-visuals-utils-interactivityutils": "0.3.1",
-    "powerbi-visuals-utils-svgutils": "1.0.0",
-    "powerbi-visuals-utils-testutils": "1.0.0",
-    "powerbi-visuals-utils-tooltiputils": "0.3.0",
-    "powerbi-visuals-utils-typeutils": "1.0.0",
+    "powerbi-visuals-utils-chartutils": "^1.0.0",
```
Comment (ignatvilesov) [inline package.json]: I think that it would be better to get rid of `^` sign

File: pbiviz.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "displayName": "Tornado",
     "guid": "TornadoChart1452517688218",
     "visualClassName": "TornadoChart",
-    "version": "0.6.4",
+    "version": "0.6.5",
```
Comment (ignatvilesov) [inline pbiviz.json]: Please use `1.0.1` instead

File: src/tornadoChart.ts [hunk 4]
```diff
@@ -390,10 +395,13 @@ module powerbi.extensibility.visual {
                 .withMeasure(queryName)
                 .createSelectionId();
 
+            let sourceGroupName = null;
```
Comment (ignatvilesov) [inline src/tornadoChart.ts]: Please specify a type

File: src/tornadoChart.ts [hunk 5]
```diff
@@ -390,10 +395,13 @@ module powerbi.extensibility.visual {
...
+            if (source.groupName !== undefined && source.groupName !== null) sourceGroupName = "" + source.groupName;
```
Comment (ignatvilesov) [inline src/tornadoChart.ts]: Please use curly brackets

File: src/tornadoChart.ts [hunk 6]
```diff
@@ -608,7 +616,12 @@ module powerbi.extensibility.visual {
         }
 
         private updateElements(): void {
-            let elementsTranslate: string = translate(this.allLabelsWidth, 0);
+            let translateX: number = 0,
+                position: string = DataViewObjectModule.getValue(this.dataView.categoriesObjectProperties, "position", legendPosition.left);
```
Comment (ignatvilesov) [inline src/tornadoChart.ts]: Can we use DataViewObjectParser to parse settings?

File: src/tornadoChart.ts [hunk 7]
```diff
@@ -687,6 +700,14 @@ module powerbi.extensibility.visual {
                     TornadoChart.DefaultTornadoChartSettings.categoriesFillColor,
                     objects,
                     colors),
+                categoriesFontSize: DataViewObjectsModule.getValue<number>(
+                    objects,
+                    tornadoChartProperties.categories.fontSize,
+                    TornadoChart.DefaultTornadoChartSettings.legendFontSize),
+                categoriesPosition: DataViewObjectModule.getValue<string>(
```
Comment (ignatvilesov) [inline src/tornadoChart.ts]: Can we use DataViewObjectParser to parse settings?

File: pbiviz.json (hunk 3 above)
Comment (ignatvilesov) [inline pbiviz.json]: Please update version in package.json as well

File: src/tornadoChart.ts (hunk 7 above)
Comment (vtkalek) [inline src/tornadoChart.ts]: no

File: src/tornadoChart.ts (hunk 6 above)
Comment (vtkalek) [inline src/tornadoChart.ts]: no

File: src/tornadoChart.ts [hunk 8]
```diff
@@ -379,7 +383,8 @@ module powerbi.extensibility.visual {
             isGrouped: boolean,
             columnGroup: DataViewValueColumnGroup,
             colors: IColorPalette): TornadoChartSeries {
-
+            if (dataView === undefined || dataView === null)
```
Comment (ignatvilesov) [inline src/tornadoChart.ts]: It'd be better to use:
```javascript
if (!dataView) {
    return;
}
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+## 0.6.5
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Please update version to 1.0.1

File: package.json [hunk 10]
```diff
@@ -36,15 +37,15 @@
...
+    "powerbi-visuals-utils-colorutils": "^1.0.0",
+    "powerbi-visuals-utils-dataviewutils": "^1.0.0",
```
Comment (AviSander) [inline package.json]: please use 1.1.0, as far as i remember it had it's dependency fixed in that version

File: pbiviz.json [hunk 11]
```diff
@@ -18,7 +18,6 @@
     "icon": "assets/icon.png"
   },
   "externalJS": [
-    "node_modules/jquery/dist/jquery.min.js",
```
Comment (ignatvilesov) [inline pbiviz.json]: Looks like it's a breaking change. Please double check it

Comment (vtkalek) [inline pbiviz.json]: fixed

**Pull request merged**